tailwind.config.js

```diff
@@ -48,6 +57,11 @@ module.exports = {
       "v-xxs": { raw: "(min-height: 300px)" },
     },
     extend: {
+      // can add back up fonts here
+      fontFamily: {
```

Comment:
do we need these in both the theme and extend objects? We should probably also put some backup fonts (ie. Sans-serif for Lato). I think google fonts, lists what the backups should be.